code/espurna/config/dependencies.h

```diff
@@ -0,0 +1,76 @@
+#ifndef DEPENDENCIES_H
+#define DEPENDENCIES_H
+
+//------------------------------------------------------------------------------
+// Do not change this file unless you know what you are doing
+// Configuration settings are in the settings.h file
+//------------------------------------------------------------------------------
+
+
+#if DEBUG_TELNET_SUPPORT
+#undef TELNET_SUPPORT
+#define TELNET_SUPPORT          1
+#endif
+
+#ifndef DEBUG_WEB_SUPPORT
+#define DEBUG_WEB_SUPPORT       WEB_SUPPORT  // Enable web debug log if web is enabled too
+#endif
+
+#if DEBUG_WEB_SUPPORT
+#undef WEB_SUPPORT
+#define WEB_SUPPORT             1           // Chicken and egg :)
+#endif
+
+#if WEB_SUPPORT == 0
```

Comment:
Preprocessor also supports direct boolean check - `#if not WEB_SUPPORT` or `#if !WEB_SUPPORT`

Reply:
done.